lib/tx_build/asset.ex

```diff
@@ -7,9 +7,10 @@ defmodule Stellar.TxBuild.Asset do
 
   @behaviour Stellar.TxBuild.XDR
 
+  @type issuer :: AccountID.t() | nil
   @type validation :: {:ok, any()} | {:error, atom()}
 
-  @type t :: %__MODULE__{code: String.t(), issuer: String.t(), type: atom()}
+  @type t :: %__MODULE__{code: String.t(), issuer: issuer() | nil, type: atom()}
```

Comment:
Is it valid to define `nil` as a part of the type declaration? If that is the case, is it required to use `nil` in line 13?

Reply:
no, it is not. It is already declared in line #10. Removing it .... thanks!